src/languageservice/services/yamlCompletion.ts

```diff
@@ -1552,11 +1553,21 @@ export class YamlCompletion {
         hasProposals = true;
       });
     }
-    this.collectDefaultSnippets(schema, separatorAfter, collector, {
+
+    let stringifySettings = {
```

Comment:
wouldn't be more easier (maybe readable) to keep old version and just write

```ts
this.collectDefaultSnippets(schema, separatorAfter, collector, {
      newLineFirst: isArray,
      indentFirstObject: isArray,
      shouldIndentWithTab: isArray,
    });
```

Reply:
I was trying to show an explicit distinction between the two. I can change it, however it will be reverse of your suggestion
```typescript
this.collectDefaultSnippets(schema, separatorAfter, collector, {
      newLineFirst: !isArray,
      indentFirstObject: !isArray,
      shouldIndentWithTab: !isArray,
    });
```